packages/now-node/src/helpers.ts

```diff
@@ -73,6 +73,18 @@ function status(res: NowResponse, statusCode: number): NowResponse {
   return res;
 }
 
+function redirect(
+  res: NowResponse,
+  url: string,
```

Comment:
This needs to match the usage in Next.js

- `res.redirect(301, '/go')`
  - should have `res.statusCode === 301 && res.headers.location === '/go'`
- `res.redirect('/go')`
  -  should have `res.statusCode === 307 && res.headers.location === '/go`

Reply:
Agreed with @styfle, we want to match the usage in Next.js, which is based on Express. You can see the [related Express docs here](http://expressjs.com/en/4x/api.html#res.redirect).